projects/word-game/words file.py

```diff
@@ -0,0 +1,18 @@
+import random
+
+def RandWords(Word_length,Number_of_words):
```

Comment:
please use `snake case(snake_case)` convention for variables and  `camel case(camelCase)`  for function names.